webpack.common.js

```diff
@@ -37,9 +37,10 @@ module.exports = {
     extensions: ['*', '.js', '.jsx'],
     alias: {
       'react-dom': '@hot-loader/react-dom',
-      utils: path.resolve(__dirname, 'src/utils/'),
-      _common: path.resolve(__dirname, 'src/components/_common/'),
-      'tapis-redux': path.resolve(__dirname, 'src/tapis-redux/'),
+      utils: path.resolve(__dirname, 'tapis-ui/utils/'),
+      _common: path.resolve(__dirname, 'tapis-ui/src/_common/'),
+      'tapis-app': path.resolve(__dirname, 'tapis-app/src/'),
```

Comment:
Since `tapis-app` is the client of `tapis-ui` (the App built on the common UI elements) then `tapis-ui` would exist here rather than `tapis-app`. Also, is there a reason why `react-dom` has to be resolved to the hot-loader in the webpack config? This could, in theory, land us in some trouble if react-dom were to be updated.